backend/app/api/routes/model_management.py

```
@@ -0,0 +1,263 @@
+
+
+@router.delete("/models/{model_name}", response_model=ModelOperationResponse)
+async def delete_model(
+    model_name: str,
+    _current_user: User = Depends(get_current_user),
+):
+    """
+    动态删除模型配置
+    """
+    try:
+        result = await _call_litellm_admin_api("POST", "/model/delete", {"id": model_name})
```

Ruff (0.11.9)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)